### ISSUE TITLE
Clarification on the training data used for R-SuperPancreasKidney

### ISSUE THREAD

**Issue by ZHN-02** [MrGiovanni/R-Super#6]
Hi,Thank you very much for releasing the R-Super models and AbdomenAtlas resources.

I am currently evaluating the Hugging Face checkpoint **AbdomenAtlas/R-SuperPancreasKidney** and would like to perform a fair independent evaluation.

According to the model card, the model was trained using:

* 2229 pancreatic tumor CT-report pairs
* 2738 kidney tumor CT-report pairs
* 344 pancreatic tumor CT-mask pairs
* 1674 kidney tumor CT-mask pairs
* 8995 control CTs

However, I could not find the exact case list or split information corresponding to these numbers.

Could you please clarify:

1. Which subset of AbdomenAtlas 3.0 was used to train R-SuperPancreasKidney?
2. Did the model use the official AbdomenAtlas 3.0 IID split, OOD split, or a different internal split?
3. Are the training case IDs available somewhere?
4. If I want to evaluate R-SuperPancreasKidney on completely unseen AbdomenAtlas cases, which cases would you recommend excluding from the evaluation because they were already used during training?

My goal is to use the remaining AbdomenAtlas 3.0 cases as an independent test set and avoid any train-test overlap.

Thank you very much for your help.

**Comment by PedroRASB** [comment]
Good afternoon, thank you for your question. We included the entire AbdomenAtlas 3.0 in training, since we did not test on AbdomenAtlas 3 in our paper. Instead, we tested on private datasets (UCSF, internal validation; Johns Hopkins Hospital, external validation). So, I would suggest you test the models on datasets not part of AbdomenAtlas 3.0, such as Panorama (pancreas tumor segmentation) or Merlin (pancreas and kidney tumor detection). For Merlin, you can get some guiding from the demo in our readme, but that demo is for pancreatic tumors, so you would just need to include kidney tumors and change the model checkpoint. Evaluation on Merlin would be fully out-of-distribution and fair, these models were trained before Merlin was released.
